src/_guides/language/effective-dart/style.md

```diff
@@ -323,6 +315,16 @@ defaultTimeout
 kDefaultTimeout
 {% endprettify %}
 
+### DON'T explicitly name libraries
+
+Appending a name to the `library` directive is technically possible,
+but is a legacy feature and discouraged. 
+
```

Comment:
Should the good and bad examples you currently have in the language tour be here?

Reply:
Yes! Wow, now I'm really convinced I just mixed up the pages I was working on haha